src/power/PowerComponentWin.h

```diff
@@ -9,6 +10,14 @@ class PowerComponentWin : public PowerComponent
   PowerComponentWin() : PowerComponent(0) { }
   virtual void doDisableScreensaver();
   virtual void doEnableScreensaver();
+
+  virtual bool canPowerOff() override { return WinUtils::getPowerManagementPrivileges(); }
```

Comment:
Hmm, do we really want to call this every time we show the dialog? Wouldn't it be better to init this once and save the result so that we know if it was successful or not?